rmw_fastrtps_cpp/src/type_support_common.cpp

```diff
@@ -129,6 +144,55 @@ bool TypeSupport::deserializeROSmessage(
   return true;
 }
 
+bool TypeSupport::get_key_hash_from_ros_message(
+  void * ros_message,
+  eprosima::fastrtps::rtps::InstanceHandle_t * ihandle,
```

Comment:
should we check if `ihandle` is not NULL before accessing it?

Reply:
I think an `assert(ihandle)` would be enough, as we are doing with `ros_message`.

@Mario-DL would you mind adding that to all the implementations of `get_key_hash_from_ros_message`?